app/pages/register/settlors/living_settlor/trust_type/SetUpInAdditionToWillTrustYesNoPage.scala

```diff
@@ -31,8 +31,12 @@ case object SetUpInAdditionToWillTrustYesNoPage extends QuestionPage[Boolean] {
 
   override def cleanup(value: Option[Boolean], userAnswers: UserAnswers): Try[UserAnswers] = {
     value match {
+      case Some(false) =>
+        userAnswers.remove(DeceasedSettlor)
       case Some(true) =>
-        userAnswers.remove(HowDeedOfVariationCreatedPage)
+        userAnswers.remove(KindOfTrustPage)
+          .flatMap(_.remove(HoldoverReliefYesNoPage))
```

Comment:
Changing answer to set up in additional to a will trust? Should not clean up the answers for holdover relief, kind of trust. You're correct on the Living, we also need to clean the 'why was the deed of variation created'?